### PR TITLE
Fix Azure AD mappings

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2667

### CODE REVIEW

File: astro/set-up-scim-provisioning.md [hunk 1]
```diff
@@ -150,16 +150,18 @@ Complete the manual setup if you configured your existing Astro app without usin
     | userPrincipalName                                           | userName        |
```
Comment (aliotta) [inline astro/set-up-scim-provisioning.md:150]: lets add the same note to delete extras for the group mappings as well

File: astro/set-up-scim-provisioning.md [hunk 2]
```diff
@@ -141,6 +141,10 @@ Complete the manual setup if you configured your existing Astro app without usin
     | displayName                      | displayName     |
     | members                          | members         |
 
+    Delete any other user attributes not listed in the previous table. You should have exactly two attributes as shown in the following screenshot:
```
Comment (aliotta) [inline astro/set-up-scim-provisioning.md:144]: user -> group

Comment (jwitz) [inline astro/set-up-scim-provisioning.md:144]:
```suggestion
    Delete any other group attributes not listed in the previous table. You should have exactly two attributes as shown in the following screenshot:
```

File: astro/set-up-scim-provisioning.md [hunk 3]
```diff
@@ -141,6 +141,10 @@ Complete the manual setup if you configured your existing Astro app without usin
...
+
+    ![Azure group mappings with only the correct 2 attributes listed](/img/docs/azure-grop-mappings.png)
```
Comment (jwitz) [inline astro/set-up-scim-provisioning.md:146]:
```suggestion
    ![Azure group mappings with only the correct 2 attributes listed](/img/docs/azure-group-mappings.png)
```

**Review by aliotta — APPROVED** [review]

**Pull request merged**